bin/add_transform_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""Automatically create transform tests input and output files given an input template.
+
+Usage:
+    add_transform_test.py --template-file=sam-template.yaml
+
+Options:
+    --template-file=<i>       Location of SAM template to transform [default: template.yaml].
+
+"""
+import re
+import os
+import shutil
+import yaml
+from docopt import docopt
+from pathlib import Path
+
+from samtranslator.yaml_helper import yaml_parse
+from sam_translate import transform_template
+
+cwd = os.getcwd()
+CLI_OPTIONS = docopt(__doc__)
+
+
+def get_input_file_path():
+    input_file_option = CLI_OPTIONS.get("--template-file")
+    return os.path.join(cwd, input_file_option)
+
+
+def copy_input_file_to_transform_test_dir(input_file_path, transform_test_input_path):
+    shutil.copyfile(input_file_path, transform_test_input_path)
+
+    replace_arn_partitions(transform_test_input_path)
+
+
+def generate_transform_test_output_files(input_file_path, file_basename):
+    output_file_option = file_basename + ".json"
+
+    # run sam_translate.py and get the temporary output file
+    temp_output_file_path = os.path.join(cwd, output_file_option)
+    transform_template(input_file_path, temp_output_file_path)
+
+    # copy the output files into correct directories
+    transform_test_output_paths = [
+        cwd + path + output_file_option
+        for path in [
+            "/tests/translator/output/",
+            "/tests/translator/output/aws-cn/",
+            "/tests/translator/output/aws-us-gov/",
+        ]
+    ]
+    for output_path in transform_test_output_paths:
```

Comment:
Would be good at least print out that the file is copied, otherwise there's no output and it could be confusing.

Reply:
Good point!